README.md

```diff
@@ -4,11 +4,11 @@
   </a>
   <h1>innerself</h1>
   <p>
-    A tiny view + state management solution using <code>innerHTML</code>.
     <a href="https://travis-ci.org/stasm/innerself">
       <img alt="Build Status" src="https://travis-ci.org/stasm/innerself.svg?branch=master" />
     </a>
   </p>
+  <p>A tiny view + state management solution using <code>innerHTML</code>.</p>
```

Comment:
Should we put the Travis badge below the description sentence?

Reply:
Most other popular git pages tend to have them directly below the icon, but it's honestly up to you. I just personally think it looks really bad next to the description.

Reply:
OK, let's merge this as you originally suggested. Thanks!